assets/apps/dashboard/src/Components/Card.js

```diff
@@ -29,7 +29,10 @@ const Card = (props) => {
 			</div>
 			<div className="card-content">
 				{description && (
-					<p className="card-description">{description}</p>
+					<p
+						className="card-description"
+						dangerouslySetInnerHTML={{ __html: description }}
+					></p>
```

Comment:
We should refrain from using dangerouslySetInnerHTML if possible. Let's do it as I mentioned below 🚀